evalml/tests/data_checks_tests/test_invalid_targets_data_check.py

```diff
@@ -81,20 +83,20 @@ def test_invalid_target_data_check_multiclass_two_examples_per_class():
     expected_message = "Target does not have at least two instances per class which is required for multiclass classification"
 
     # with 1 class not having min 2 instances
-    assert invalid_targets_check.validate(X, y=pd.Series([0, 1, 1, 2, 2])) == {
+    assert invalid_targets_check.validate(X, y=pd.Series([0] + [1] * 19 + [2] * 80)) == {
```

Comment:
This is an edit to make the targets pass the test that looks for targets with large numbers of unique values relative to the total number of target values.  I do this in a few places.